index.html

```diff
@@ -15,31 +15,294 @@
 </head>
 
 <body>
-    <header class="navbar container fixed-top">
-        <a href="" id="logo">
+    <div class="container">
+        <header class="navbar container fixed-top">
+            <a href="" id="logo">
             Welcome
         </a>
-        <nav class="menu">
-            <i class="fa-solid fa-bars "></i>
-        </nav>
-    </header>
-    <section class="header">
-        <div class="container">
-            <h1 class="identity">
-                Hey there. I'm, <br/> Tite Lunda <br><span>I'm a software developer</span>
-            </h1>
-            <p class="profile">
-                <b></b> I can help you build a product , feature or website Look through some of my work and experience! If you like <br/>what you see and have a project you need coded, don’t hestiate to contact me. </p>
-
-            <ul class="icon-group">
-                <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-brands fa-linkedin"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
-                <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
-            </ul>
+            <nav class="menu">
+                <i class="fa-solid fa-bars "></i>
+            </nav>
+        </header>
+
+        <section class="header">
+            <div class="container">
+                <h1 class="identity">
+                    Hey there. I'm, <br/> Tite Lunda <br><span>I'm a software developer</span>
+                </h1>
+                <p class="profile">
+                    <b></b> I can help you build a product , feature or website Look through some of my work and experience! If you like <br/>what you see and have a project you need coded, don’t hestiate to contact me. </p>
+
+                <ul class="icon-group">
+                    <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-brands fa-linkedin"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
+                    <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
+                </ul>
+                <ul class="icon-group">
+                    <li><i class="fa-solid fa-chevron-down">
+                    </i>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="works">
+            <div class="Recent">
+                <h1>My Recents Works</h1>
```

Comment:
- [ ] There should be only one `h1` in an HTML file because of semantics and it helps search engines and web crawlers easily understand the main subject of your page. I kindly suggest that you make only your main title of " Hey there. I'm, Tite Lunda I'm a software developer" an `h1` and kindly make use of headers from `h2` down to `h6` for any other headers in your HTML file.
- [ ] please kindly make changes and replace all other `h1` except your main title